README.md

```diff
@@ -619,6 +620,33 @@ For versions of ISIS prior to ISIS 4.1.0, please cd into `$ISIS3DATA` instead of
     rsync -azv --delete --partial isisdist.astrogeology.usgs.gov::isisdata/data/voyager1 .
     rsync -azv --delete --partial isisdist.astrogeology.usgs.gov::isisdata/data/voyager2 .
 
+### ISIS Test Data 
+ISIS is comprised of two types of tests, custom Makefile based tests, and GTest based tests. Those that are GTest based, make economical use of data that exists on the ISIS3 repo along with the source, so no special data is required to run those other than the ISIS data area. The Makefile tests depends on a separate source of data that consists of a few gigabytes of input and expected output data used for testing ISIS applications. The total size of this test data decreases as we work towards converting Makefile tests to GTests.  
+ 
+###How to download the ISIS test data with rclone  
+Test data is hosted using Amazon S3 storage buckets. We recommend using rclone to pull the data into a local directory. You can download rclone using their instructions (see: https://rclone.org/downloads/) or by using an anaconda environment (see: https://docs.anaconda.com/anaconda/install/). If you already have an anaconda environment up, install rclone with: conda install –c conda-forge rclone  
+
+Next, you will want to configure rclone using a default S3 configuration. See: https://rclone.org/s3/ for detailed information on how to configure S3, but for the purposes of downloading the ISIS3 test data, you simply run rclone config which will start an interactive menu. Press enter through it all except for these details: 
+
+1. Set S3 as both your storage type and storage provider
+2. Set us-west-2 as both your region to connect to and as the location constraint. 
+3. Everything else, just leave as the default. 
+
+Example output: https://gist.github.com/Kelvinrr/706bbd54b1c2c30d0ce3d12f7dcaa10a
+
+Once rclone is configured, simply run: `rclone sync remote:asc-isisdata/isis_testData/ $ISISTESTDATA`
+where:
+  - $ISISTESTDATA is the directory you want to download the test data to. Whatever directory you use, you want to set the environment variable $ISISTESTDATA to this directory
```

Comment:
Add something under the ISIS Test Data header indicating that the makefile tests find the test data using the `ISISTESTDATA` environment variable. Then, you can remove this bullet.